Presentation.Web/Controllers/API/V1/OData/AdviceController.cs

```diff
@@ -108,7 +108,7 @@ public override IHttpActionResult Post(int organizationId, Advice advice)
                 //Advice cannot be an orphan - it must belong to a root
                 return BadRequest($"Both {nameof(advice.RelationId)} AND {nameof(advice.Type)} MUST be defined");
             }
-            if (advice.Reciepients.Where(x => x.RecpientType == RecieverType.USER).Any(x => !_emailValidationRegex.IsMatch(x.Name)))
+            if (advice.Reciepients.Where(x => x.RecpientType == RecieverType.USER).Any(x => !_emailValidationRegex.IsMatch(x.Email)))
```

Comment:
think the receivrtype.USER is a bit wierd.. we should rename it to SPECIFIC_EMAIL